### DISCUSSION TITLE
Starting Nginx  module  failed in ansible

### DISCUSSION THREAD

**Discussion by chowdary709** [learndevopsonline/learndevopsonline#1872] [Q&A]
![Screenshot (12)](https://github.com/learndevopsonline/learndevopsonline/assets/149947513/e475774e-3110-468e-8816-d9989219c97c)
![Screenshot (13)](https://github.com/learndevopsonline/learndevopsonline/assets/149947513/61594278-b9d0-482f-a808-98c00b7164c8)
![Screenshot (14)](https://github.com/learndevopsonline/learndevopsonline/assets/149947513/392ea5c9-b760-4c3d-958e-5c5a1f9031d6)

**Comment by chaithanya1812** [comment]
Hi @chowdary709, I am unable to login into server,
so will please join lab session

**Reply by vravru135** [reply]
@chowdary709  i'm also getting same error . can u share ur answer pls

**Reply by chaithanya1812** [reply]
hello @vravru135 , join lab session once

https://learndevopsonline.my.canva.site/lab-help

**Reply by vravru135** [reply]
@chaithanya1812 okay ...thank you

**Reply by chowdary709** [reply]
@vravru135   meru  expense.conf file lo backend dns name istay backend instance run lo undai and dms record attach chasiundali then working.

**Reply by vravru135** [reply]
@chowdary709  yeah i have cleared that....thanks

**Comment by chaithanya1812** [comment]
Hello @chowdary709 & @vravru135 , 
1) You have doubts related to starting nginx service, if it's not cleared please join lab session
2) Next time after posting an error try to make sure that your instances upon running, Don't stop & terminate the instances

**Reply by chowdary709** [reply]
ok done @chaithanya1812
